custom.css

```
@@ -173,6 +174,11 @@ html[data-theme="dark"] :not(pre):not(mark) > code {
   padding: 0 !important;
 }
 
+html[data-theme="dark"] .content mark{
+  background:var(--yellow-highlight);
+  color:var(--base2)
+}
+
```

Small nit, can we format this?

```suggestion
html[data-theme="dark"] .content mark {
  background: var(--yellow-highlight);
  color: var(--base2);
}
```